data/cameras.xml

```diff
@@ -6284,29 +6206,8 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
-	<Camera make="NIKON" model="E8800" mode="12bit-compressed" decoder_version="3" supported="no">
-		<ID make="Nikon" model="Coolpix E8800">Nikon Coolpix E8800</ID>
-		<CFA width="2" height="2">
-			<Color x="0" y="0">BLUE</Color>
-			<Color x="1" y="0">GREEN</Color>
-			<Color x="0" y="1">GREEN</Color>
-			<Color x="1" y="1">RED</Color>
-		</CFA>
-		<Crop x="0" y="0" width="3280" height="2454"/>
-		<Sensor black="0" white="4095"/>
-		<Hints>
-			<Hint name="coolpixsplit" value=""/>
-		</Hints>
-		<ColorMatrices>
-			<ColorMatrix planes="3">
-				<ColorMatrixRow plane="0">7971 -2314 -913</ColorMatrixRow>
-				<ColorMatrixRow plane="1">-8451 15762 2894</ColorMatrixRow>
-				<ColorMatrixRow plane="2">-1442 1520 7610</ColorMatrixRow>
-			</ColorMatrix>
-		</ColorMatrices>
-	</Camera>
-	<Camera make="NIKON" model="E8800" mode="12bit-uncompressed" decoder_version="3" supported="no">
```

Comment:
Git is a wonderful tool. Learn to use it, and it will answer many questions that should not be left to the reviewer, but be explained in the commit message. Really.

This was due to
```
commit 6eb86833b29bfb7972db2a0f3f8d3d9f024f8dac
Author:	Roman Lebedev <lebedev.ri@gmail.com>  Mon Sep 12 21:39:01 2016
Committer:	Roman Lebedev <lebedev.ri@gmail.com>  Fri Nov  4 17:57:41 2016

Cameras.xml: disable all nikon cameras with questionable whitelevel.

Ugh.

If you are affected by this, you *need* to report this to the
https://redmine.darktable.org/projects/darktable/issues/new
and provide one raw file per each camera's raw mode.
```